packages/js-commons/src/core/local-files.ts

```diff
@@ -167,8 +177,10 @@ export const saveArtifact = async (
   options: {
     relativePath?: string;
     skipMetadata?: boolean;
+    skipContent?: boolean;
   } = {
     skipMetadata: false,
+    skipContent: false,
```

Comment:
Don't add this. This function is only for writing the artifact. Create a separate function for saving the file contents to the system.

Reply:
If the file and the artifact are separate writes then this shouldn't happen. The EnsembleDocument of Asset/Font has no content just a reference to the location of the actual file.